production/scrapers/michigan_vaccine.R

```diff
@@ -9,8 +9,9 @@ michigan_vaccine_date_check <- function(x, date = Sys.Date()){
         rvest::html_text() %>% 
         {.[str_detect(., "(?i)correction")]} %>% 
         stringr::str_split("(?i)through") %>% 
```

Comment:
I think both this line and the line below are unnecessary for the current date format. It still works with them included, though!